### ISSUE TITLE
NVDA announces selected for every cell when UIA is enabled in Microsoft Excel

### ISSUE THREAD

**Issue by leonardder** [nvaccess/nvda#12530]
### Steps to reproduce:
1. Ensure that UIA for Excel is enabled
2. Open Excel
3. Navigate through the sheet

### Actual behavior:
Nvda announces the word selected for every cell

### Expected behavior:
NVDA does not announce selected for every cell.

### Underlying cause
getSelectedItemsCount on the selection container returns 0 even though the focused item is selected.

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
alpha-23054,43f4b192

#### Windows version:
Windows 10 21H1

#### Name and version of other software in use when reproducing the issue:
Excel for Office365 version 16.0.14026.20246

**Comment by leonardder** [comment]
@michaelDCurran I don't recall this behavior from earlier tests with Excel, so this might be a regression.
